src/index.html

```diff
@@ -5,8 +5,401 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__link-logo" href="#home">
+        <img
+          class="logo"
```

Comment:
if I am not mistaken, it's not according to the BEM

Reply:
if you want to make the logo a separate BEM block, you can use the `logo` class, but in this case it is better to make the logo an element of the header  - `header__logo`.